docs/walkthrough.rst

```diff
@@ -0,0 +1,199 @@
+(do not confiuse with the option of exporting tags). This grants additional  \
+flexibility, either to select objects attached to different projects \
+or different owners, or to have a finer control over \
+which objects to process.
+
+.. figure:: images/annotation_03_selecting_tags.png
+
+   *Selection of a subset of datasets from a tag and the auto-filled Export to CSV script.*
+..
+
+The selection works in the same way as for the children selection. Note \
+that choosing "*<selected>*" for "*Target Data Type*" will result in an \
+error in this case.
+
+All from selected
+^^^^^^^^^^^^^^^^^
+The fourth option is to apply the scripts to all of the selected objects and the children  \
+objects. Selecting a Project and using the option **<all (from selected)>** with result in \
+applying the scripts to the selected project, to the linked datasets and the linked images \
+bellow. Note that this option does not exist for the import of annotation from csv.
+
+Exporting annotations
+---------------------
+
+Exporting annotations will generate a .csv file with the header names \
+expected by the import .csv script. This has two use:
+
+* export existing key-value pairs (along other things like namespace or tags).
+  This can be useful to modify them before re-import, or to transfer annotations
+  across groups.
+* export a list of objects with their name and ID.
+
+.. figure:: images/annotation_04_export_output.png
+
+   *Exported .csv with only the object ID and name*
+..
+
+  Tip: If you have Key-Value pairs attached to your objects that you do not \
+  wish to export (to create a template like shown above), specify a not-in-use \
+  namespace (any random input will work). This does not affect tags.
+
+
+Importing annotations
+---------------------
+
+Starting from the file we exported as explained in the previous section, \
+we proceed to edit it within a spreadsheet editor, adding more columns to the csv, \
+including a row specifying the namespace to assign for each key:
+
+.. figure:: images/annotation_05_KV_to_import.png
+
+   *Result of populating the csv shown in a text editor*
+..
+
+We added several columns to annotate our dataset with Key-Value pairs \
+following the `REMBI <https://doi.org/10.1038/s41592-021-01166-8>`_ guidelines \
+(after saving the document, our ``TAB`` separator became ``,``).
+
+We proceed and start the script "*Import from CSV*".
+
+.. figure:: images/annotation_06_script_import.png
+
+   *Selection of all the datasets of a project and the Import from CSV script.*
+..
+
+There are many parameters here that could help you fine tune the way to import annotations. More \
+  on that in the :doc:`parameter description section</parameter_description>` of this guide.
+
+
+We can then see in the OMERO activities that the Key-Value pairs were added to 5 \
+datasets out of the 11 present in this project (as expected). We also see three set of \
+key-value pairs, one for each namespace we used in the csv.
```

Comment:
```suggestion
Key-Value pairs, one for each namespace we used in the csv.
```

I actually do not understand. The namespace usage comes as a big surprise for me. Can you introduce more slowly ?

Reply:
(maybe bearing in mind that most of the users do not know and do not want to know about what namespace even is...)